run_pcap.go

```diff
@@ -101,12 +109,38 @@ func NewFlow(pkt gopacket.Packet) *Flow {
 }
 
 func runPcap(c *cli.Context) error {
+
+	if c.String("strategy") != "" && c.String("strategyFile") != "" {
+		return cli.Exit("strategy and strategyFile are mutually exclusive", 1)
+	}
+
 	input := c.String("input")
 	output := c.String("output")
-	strat := c.Args().First()
+	var stratString string
+
+	stratString = c.String("strategy")
+	if stratString == "" {
+		strategyFile := c.String("strategyFile")
+		if strategyFile == "" {
+			errStr := "must provide one of -strategy or -strategyFile"
+			logger.Error(errStr)
+			return cli.Exit(errStr, 1)
+		}
+
+		in, err := os.ReadFile(strategyFile)
+		if err != nil {
+
+			errStr := fmt.Sprintf("cannot open %s: %v", strategyFile, err)
+
+			logger.Error(errStr)
+			return cli.Exit(errStr, 1)
+		}
+		stratString = string(in)
+	}
 
-	s, err := geneva.NewStrategy(strat)
+	strat, err := geneva.NewStrategy(stratString)
 	if err != nil {
+		fmt.Printf("invalid strategy: %s", err)
```

Comment:
What's with this random print statement?

Reply:
It was so when I run it from the python script I could see what the error was